include/SugarObjects/templates/person/language/en_us.lang.php

```diff
@@ -72,9 +72,12 @@
     'LBL_ALT_ADDRESS_STATE' => 'Alternate State',
     'LBL_ALT_ADDRESS_POSTALCODE' => 'Alternate Postal Code',
     'LBL_ALT_ADDRESS_COUNTRY' => 'Alternate Country',
+    'LBL_PRIMARY_STREET' => 'Address',
+    'LBL_ALT_STREET' => 'Other Address',
     'LBL_STREET' => 'Other Address',
     'LBL_CITY' => 'City',
     'LBL_STATE' => 'State',
+    'LBL_POSTALCODE' => 'Postal Code',
```

Comment:
we will end up with 2 similar strings with a small variation on the key. Maybe cleaning up the code to just have on of them?
```
    'LBL_POSTALCODE' => 'Postal Code',
    'LBL_POSTAL_CODE' => 'Postal Code',
```